draft-ietf-jose-pqc-kem.md

```diff
@@ -372,11 +369,11 @@ ML-KEM-1024 MUST be used with a KDF capable of outputting a key with at least 25
  +=================================+===================================+
  | alg                             | Description                       |
  +=================================+===================================+
- | MLKEM512+A128KW                 | ML-KEM-512 + AES128KW             |
+ | ML-KEM-512+A128KW               | ML-KEM-512  + AES128KW            |
  +=================================+===================================+
- | MLKEM768+A192KW                 | ML-KEM-768 + AES192KW             |
+ | ML-KEM-768+A192KW               | ML-KEM-768  + AES192KW            |
```

Comment:
```suggestion
 | ML-KEM-512+A128KW               | ML-KEM-512 + AES128KW            |
+=================================+===================================+
 | ML-KEM-768+A192KW               | ML-KEM-768 + AES192KW            |
```

Reply:
I am unable to commit the suggestion, I get the error "Applying suggestions on deleted lines is currently not supported".

Reply:
Right, just remove the extra space before the + signs (there are two in some entries).

Reply:
Fixed.